metriq_gym/exporters/base_exporter.py

```diff
@@ -0,0 +1,25 @@
+from abc import ABC, abstractmethod
+import importlib.metadata
+from metriq_gym.benchmarks.benchmark import BenchmarkResult
+from metriq_gym.job_manager import MetriqGymJob
+
+
+class BaseExporter(ABC):
+    def __init__(self, metriq_gym_job: MetriqGymJob, results: BenchmarkResult):
+        self.metriq_gym_job = metriq_gym_job
+        self.results = results
+        super().__init__()
+
+    def _as_dict(self):
```

Comment:
Should we consider making this a property and using `as_dict` instead? That is:

```suggestion
    @property
    def as_dict(self):
```

It's fine as is, but maybe this is a bit more explicit?